modules/utility/utility.py

```diff
@@ -4,12 +4,14 @@
 logger = logging.getLogger(__name__)
 
 
-def set_key_from_env(conf, key, env_name):
-    try:
-        value = os.environ[env_name]
-    except KeyError:
-        return
-
+def set_key_from_env(conf, key, env_name, default=None):
```

Comment:
Do we use this function after the changes?

Reply:
Nope